src/Filter/AbstractDateFilter.php

```diff
@@ -119,26 +124,41 @@ public function getRenderSettings()
         ]];
     }
 
+    abstract protected function applyTypeIsLessEqual(ProxyQueryInterface $queryBuilder, string $field, array $data);
```

Comment:
AFAIK this is a BC break, because you introduce new API methods without that MUST be implemented.

Reply:
See https://github.com/sonata-project/SonataDoctrineMongoDBAdminBundle/pull/368/files#r443126771